docs/en/MockFunctionAPI.md

```diff
@@ -138,13 +138,32 @@ console.log(myMockFn(), myMockFn(), myMockFn(), myMockFn());
 > 'first call', 'second call', 'default', 'default'
 ```
 
-### `mockFn.mockReturnThis()`
```

Comment:
why is this removed?

Reply:
Looks like an accident on my part. I'll fix it.